Rocket.Chat/Extensions/StringExtensions.swift

```diff
@@ -96,7 +96,11 @@ extension String {
     }
 
     func removingWhitespaces() -> String {
-        return components(separatedBy: .whitespaces).joined()
+        return components(separatedBy: .whitespacesAndNewlines).joined()
+    }
+
+    func removingNewLines() -> String {
+        return components(separatedBy: .newlines).joined()
```

Comment:
Can you add tests for this method in StringExtensionSpec.swift?

Example: string with spaces and newlines, but only newlines are removed

Reply:
✅